lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -317,6 +317,10 @@ mapping:
           used if you experience long waiting times before a job is actually submitted
           to your cluster.
 
+          This only affects tools where some requirements can be resolved but not others,
+          most modern best practice tools can use prebuilt environments in the Conda
+          directory.
```

Comment:
💯 -- and we should probably announce deprecation. This was a hack when we didn't have the all-in-one resolution yet. In the context of conda this probably makes things worse if it is actually used.